library/java/io/envoyproxy/envoymobile/engine/types/EnvoyFinalStreamIntel.java

```diff
@@ -51,9 +51,9 @@
    */
   public long getResponseStartMs();
   /*
-   * The time the last byte of the request was received, in ms since the epoch.
+   * The time when the stream reached a final state (Error, Cancel, Success), in ms since the epoch.
```

Comment:
while we're in here, did your last PR correct to unix epoch or are all epochs here the local clock epoch?  If the later would you be willing to comment at the top of each intel block what epoch is because I default to unix.

Reply:
There is only one definition of Epoch: 1970:01:01?00:00:00GMT.

The "Tick Clock" of a computer, what you named "local clock", is almost never based on Epoch. Typically, the Tick Clock starts at zero when a computer boots up. Trying to "extract" an Epoch timestamp from a Tick Clock is non-sensical. The sad reality is we do have code like this: `monotonicTime().time_since_epoch()`. This makes no sense, but that's used as a trick to retrieve the "long" value representing the number of Ticks.